modules/fedimint-ln-client/src/lib.rs

```diff
@@ -846,6 +870,14 @@ impl LightningClientModule {
         }
     }
 
+    async fn get_next_incoming_contract_child_id(
```

Comment:
Similar to the wallet tweak key, I decided to keep an index for the `payment_keypair` since the public key of this is used as the preimage to the invoice, but this is also used to spend from the contract. We don't want to use the same key here since we want a unique preimage for each contract.